Makefile

```diff
@@ -46,6 +46,8 @@ BUILD_GOPROXY ?= https://proxy.golang.org
 # LOCALIP as an env var before running 'make local' will solve that.
 LOCALIP ?= $(shell ifconfig | grep inet | grep -v '::' | grep -v 127.0.0.1 | head -n1 | awk '{print $$2}')
 
+DOCKERIP ?= $(shell docker network inspect kind | jq -r ".[0].IPAM.Config[0].Gateway")
```

Comment:
This isn't used anywhere in this changeset?

Reply:
ah, thanks, I'll pull it out. It will be in the next one instead.